bucket/w10privacy.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "3.8.0.0",
+    "description": "Adjust Windows according to your preferences to improve the performance and privacy of the OS.",
+    "homepage": "https://www.winprivacy.de/english-home",
+    "license": {
+        "identifier": "CC-BY-NC-ND-4.0",
+        "url": "https://www.winprivacy.de/deutsch-start/download"
+    },
+    "url": "https://www.winprivacy.de/app/download/12302828636/W10Privacy.zip",
+    "hash": "a14d120e34834e49b7fd399f76fbfbeb400e936b3d8b82d96bfddb047a5ca1d1",
+    "installer": {
+        "script": [
+            "if (-not (Test-Path \"$persist_dir\\W10Privacy.ini\")) { Set-Content \"$dir\\W10Privacy.ini\" '' -Encoding Ascii }",
+            "(Get-ChildItem $dir 'W10*Setup.exe').FullName | Expand-7zipArchive -DestinationPath $dir -Removal",
+            "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstal*\" -Recurse"
+        ]
+    },
+    "shortcuts": [
+        [
+            "W10Privacy.exe",
+            "W10Privacy"
+        ]
+    ],
+    "persist": "W10Privacy.ini",
+    "checkver": {
+        "url": "https://www.winprivacy.de/deutsch-start/download",
+        "regex": "(?sm)current version: v\\.([\\d.]+).*href=\"/app/download/(?<cosi>\\d+)/W10Privacy.zip"
+    },
+    "autoupdate": {
+        "url": "https://www.winprivacy.de/app/download/$matchCosi/W10Privacy.zip"
```

Comment:
Same as above for `matchCosi`

Reply:
To be honest when looking at the resulting download URL I'm not sure how to change it to something else, it's seemingly just a row of numbers;

https://www.w10privacy.de/app/download/12302828636/W10Privacy.zip?t=1633760316